### DISCUSSION TITLE
Music assistant / sqeezelite

### DISCUSSION THREAD

**Discussion by jsapede** [tteck/Proxmox#3214] [Ideas]
As installation rely on homeassistant Addon or docker with privilèges and Host network settings, maybe it would be interesting having bare installation on lxc 

made a try of the docker compose version and has huge drawbacks on network connexion  with lxc/docker networks. Cant made it interact correctly with HA with docker install on a docker-on-lxc as HAOS is on another VM. works on HAOS addon

definitely a well formed lxc would be usefull

**Comment by z4rn0x** [comment]
Yes please please please, music assistant LXC!

**Comment by EmmaDuch** [comment]
An archive script exists on the github of StevenSeifried inspired by tteck scripts.
It works, but with a logitechmediaserver_8.2.0_amd64.deb.
I installed it and updated to a 8.5.3 and reduced the partition to 10G and no problems for more than 1 year.
A new script would be welcome.
